.github/workflows/helm-chart-ci.yaml

```diff
@@ -33,6 +33,31 @@ jobs:
       - name: Verify Docs updated
         run: ./helm-docs.sh
 
+  ensure-image-overrides:
+    runs-on: ubuntu-22.04
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3.5.0
```

Comment:
```suggestion
```

Just rename the job at line 26 to `checks` or something like that.

This saves a clone for this very lightweight operation.

Reply:
done